detection_rules/schemas/definitions.py

```diff
@@ -44,24 +44,31 @@
 }
 
 
+def string_only_list(obj: List[str]):
+    """Validate all records in a list are strings."""
+    return all([isinstance(r, str) for r in obj])
+
+
```

Comment:
can probably remove this with the updated NewType declaration
```suggestion
```

Reply:
It is used still for validation as part of `StringListMaxSizeThree`

Reply:
not anymore?